src/pages/workspace/WorkspaceMoreFeaturesPage.tsx

```diff
@@ -0,0 +1,200 @@
+import type {StackScreenProps} from '@react-navigation/stack';
+import React from 'react';
+import {View} from 'react-native';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import * as Illustrations from '@components/Icon/Illustrations';
+import ScreenWrapper from '@components/ScreenWrapper';
+import ScrollView from '@components/ScrollView';
+import Section from '@components/Section';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import useWindowDimensions from '@hooks/useWindowDimensions';
+import type {CentralPaneNavigatorParamList} from '@libs/Navigation/types';
+import * as Policy from '@userActions/Policy';
+import type {TranslationPaths} from '@src/languages/types';
+import type SCREENS from '@src/SCREENS';
+import type {PendingAction} from '@src/types/onyx/OnyxCommon';
+import type IconAsset from '@src/types/utils/IconAsset';
+import AdminPolicyAccessOrNotFoundWrapper from './AdminPolicyAccessOrNotFoundWrapper';
+import PaidPolicyAccessOrNotFoundWrapper from './PaidPolicyAccessOrNotFoundWrapper';
+import type {WithPolicyAndFullscreenLoadingProps} from './withPolicyAndFullscreenLoading';
+import withPolicyAndFullscreenLoading from './withPolicyAndFullscreenLoading';
+import ToggleSettingOptionRow from './workflows/ToggleSettingsOptionRow';
+
+type WorkspaceMoreFeaturesPageProps = WithPolicyAndFullscreenLoadingProps & StackScreenProps<CentralPaneNavigatorParamList, typeof SCREENS.WORKSPACE.CATEGORIES>;
+
+type Item = {
+    icon: IconAsset;
+    titleTranslationKey: TranslationPaths;
+    subtitleTranslationKey: TranslationPaths;
+    isActive: boolean;
+    action: (isEnabled: boolean) => void;
+    pendingAction: PendingAction | undefined;
```

Comment:
I think we wanna use pendingField

Reply:
I see this is resolved but `pendingAction` is still used

Reply:
+1

Reply:
@rezkiy37 updated the pendingAction to pendingFields in onyx, this property is for the component, where `pendingAction` makes more sense